recipes/wdiff/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "wdiff" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 34ff698c870c87e6e47a838eeaaae729fa73349139fc8db12211d2a22b78af6b
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - texinfo
+  host:
+    - ncurses  # [linux]
+  run:
+    - ncurses  # [linux]
+
+test:
+  commands:
+    - "wdiff --version"
+
+about:
+  home: https://www.gnu.org/software/wdiff/
+  license: GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE.txt
```

This is already packaged in the source. Would grab it from there

```suggestion
  license_file: COPYING
```